### ISSUE TITLE
When you misspecify the model name, you don't get an error

### ISSUE THREAD

**Issue by KennethEnevoldsen** [saattrupdan/ScandEval#34]
When you misspecify the model name, you don't get an error, but it simply runs. This is naturally a user error but it would be nice if it stated that it couldn't load the model.

```bash
$ scandeval -m this_is_not_a_HF_model
2022-06-06 10:35:07,485 [INFO] <scandeval.benchmark>
↳ Updating the list of benchmark datasets
Downloading builder script: 6.33kB [00:00, 3.00MB/s]                                                                                                                                                                                                                                                                  
Downloading builder script: 5.27kB [00:00, 4.48MB/s]                                                                                                                                                                                                                                                                  
2022-06-06 10:35:23,881 [INFO] <scandeval.benchmark>
↳ Benchmarking this_is_not_a_HF_model on DaNE:
2022-06-06 10:35:24,890 [INFO] <scandeval.benchmark>
↳ this_is_not_a_HF_model could not be benchmarked on DaNE. Skipping.
[...]
```

Scandeval Version: 3.0.0

**Comment by saattrupdan** [comment]
I'm hesitant at throwing an error, because that could also be caused by Hugging Face's server being down, which happens quite a lot. But I can output a more descriptive message rather than the current catch-all default message.

**Comment by KennethEnevoldsen** [comment]
Potentially you can add a flag for raising an error.

**Comment by saattrupdan** [comment]
Fixed in [v4.0.0](https://github.com/saattrupdan/ScandEval/releases/tag/v4.0.0), via a `--raise-error-on-invalid-model` flag.
